src/server/httpServer.js

```diff
@@ -2,6 +2,11 @@
 const express = require('express');
 const next = require('next');
 const open = require('opn');
+const http = require('http');
```

Comment:
Is `http` module mandatory for socketio + express ?

Reply:
According to [getting started socket.io](https://socket.io/get-started/chat/), it is indeed the recommended method to use express and socket.io together. this method allows socket.io to use the same port as express